modules/openxAnalyticsAdapter.js

```diff
@@ -237,6 +244,106 @@ function removeads(info) {
   }
 }
 
+function getAuctionIdByAdId(adId) {
+
+  let auctionId;
+  utils._map(eventStack, value => value).forEach( function(auctionInfo) {
+    if(auctionInfo && auctionInfo.events){
+      let bidWonEvent;
+      bidWonEvent = auctionInfo.events.filter(function(eventsInfo) {
+        return eventsInfo.eventType === "bidWon";
+      });
+
+      if(bidWonEvent.length > 0) {
```

Comment:
FYI: don't have to do a check here, just chain them for happy fun times.  forEach on an emtpy list is just a no-op